packages/tasit-action/src/DecentralandEstate.test.js

```diff
@@ -0,0 +1,12 @@
+import DecentralandEstate from "./DecentralandEstate";
+
+import { local as localAddresses } from "../../tasit-contracts/decentraland/addresses";
```

Comment:
Good candidates for a named package.json and importing using an `@name`

Reply:
OK

Reply:
Seems that using package.json files to create "submodules" is a `react` feature.
Some refs:
https://stackoverflow.com/questions/49241941/react-native-absolute-import-with-alias-name-in-package-json
http://davidboyne.co.uk/2016/04/29/react-webpack-gem.html
https://medium.com/@sherryhsu/how-to-change-relative-paths-to-absolute-paths-for-imports-32ba6cce18a5
https://github.com/entwicklerstube/babel-plugin-root-import
https://www.npmjs.com/package/app-module-path
https://www.npmjs.com/package/babel-plugin-relative-path-import

I can keep looking but utils now, this kind of method that works with node/babel need module-directory mapping.

Reply:
Ah right - okay, you can ignore this suggestion

Reply:
If you agree, feel free to resolve this